cypress/integration/registration.spec.js

```diff
@@ -253,4 +302,27 @@ context('Actions', () => {
     cy.get(`[data-testid=thumbnail-${mockDefaultResource.identifier}]`).should('exist');
     cy.get('Button.uppy-StatusBar-actionBtn--retry').should('exist'); //because it is failing with mock
   });
+
+  it('register keyword tags', () => {
+    const testLink = 'http://www.test.com';
```

Comment:
Kanskje også en pitteliten sjekk på at det går an å fjerne tags også?

Reply:
linje 319 : cy.get('[data-testid=tag-chip-0] .MuiChip-deleteIcon').click();

Reply:
Ahh, så ikke den beklager

Reply:
synes det er greit å samle flere (tilhørende) funksjoner i samme test - da blir det ikke alt for mye overhead